yaks/README.md

```diff
@@ -665,11 +678,42 @@ yaks.call(array_of_widgets, mapper: MyCollectionMapper, item_mapper: WidgetMappe
 ```
 
 If the mapper is left unspecified, Yaks will inspect whatever you pass
-it, and try various constant lookups based on naming. These all happen
+it. First it will test the given object against the mapping defined using `mapper_for`.
+If no mapper is found, it wil call `derive_mapper_from_item` or `derive_mapper_from_collection`
+depending on whether the given object is a collection or not. If the object responds
+to `to_ary` it is considered a collection.
+
+Notes:
+* you may override one of these two methods independently from the other.
+* this function takes either a symbol or any instance
```

Comment:
As I understood, `derive_mapper_from_object` doesn't accept Symbols, only `derive_mapper_from_association`, right? And this note is for `deriver_mapper_from_object`. I don't actually think comments I mentioned are necessary.

Now when I think about it, did you add the `mapper_for` check for associations? `mapper_for` accepts Symbols so that you can specify associations, right?